src/app/modules/core/advertisement/components/card/card-top/score/score.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core'
+
+@Component({
+  selector: 'cav-score',
+  templateUrl: './score.component.html',
+  styleUrls: ['./score.component.scss'],
+})
+export class ScoreComponent implements OnInit {
+  @Input() rating = 2
+  @Input() itemId: number
+  @Output() ratingClick: EventEmitter<any> = new EventEmitter<any>()
```

Comment:
Puoi mettere direttamente @Output() ratingClick = new EventEmitter<any>()
ma il tipo any?

Reply:
Come dice Ludovico, si puo' usare il tipo implicito in questo ed ometterlo dalla dichiarazione in quanto l'assegnamento consente a TypeScript di inferire il tipo di dato.

`any` non va utilizzato mai o quasi mai e questo non e' uno dei casi per il quasi :D, per favore specificate il tipo generico the l'EventEmeitter emettera'. Grazie.